src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/SendAsyncWinHttpMockHandler.cs

```diff
@@ -4,7 +4,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace System.Net.Http.WinHttpHandlerTests
+namespace System.Net.Http.WinHttpHandlerFunctional.Tests
```

Comment:
I'm going to add a comment about this since more ppl will ask about it in the future.